Dockerfile-msphpsql

```diff
@@ -74,9 +79,6 @@ RUN sed -i -e 's/TARGET_DATABASE/msphpsql_sqlsrv/g' MsSetup.inc
 RUN sed -i -e 's/TARGET_USERNAME/'"$TEST_PHP_SQL_UID"'/g' MsSetup.inc
 RUN sed -i -e 's/TARGET_PASSWORD/'"$TEST_PHP_SQL_PWD"'/g' MsSetup.inc
 
-WORKDIR $PHPSQLDIR
-RUN chmod +x ./entrypoint.sh
-CMD /bin/bash ./entrypoint.sh
```

Comment:
the deleted lines are actually the latest dev branch, Docker file should be the latest on dev